.codebuild/canary_workflow.yml

```diff
@@ -0,0 +1,59 @@
+ version: 0.2
+env:
+  shell: bash
+  compute-type: BUILD_GENERAL1_LARGE
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: build_linux
+      buildspec: .codebuild/build_linux.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+    - identifier: test
+      buildspec: .codebuild/test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+      depend-on:
+        - build_linux
+    - identifier: publish_to_local_registry
+      buildspec: .codebuild/publish_to_local_registry.yml
+      env:
+        compute-type: BUILD_GENERAL1_MEDIUM
+      depend-on:
+        - build_linux
+    - identifier: build_app_swift_us_east_1
+      buildspec: .codebuild/run_ios_modelgen_e2e_test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TEST_SUITE: src/__tests__/build-app-swift.test.ts
+          CLI_REGION: us-east-1
+      debug-session: true
```

Comment:
here and below, why `debug-session: true`?

Reply:
In the `codegen e2e workflow` the `debug -session:  true` was present so I replicated the same Do you recommend removing it?

Reply:
Yes because canaries are run out of band from typical CI workflows triggered on PR / commit.

Including the flag here doesn't provide value and is surprising / confusing to see in a canary workflow.